recipes/org-habit-stats

```diff
@@ -0,0 +1,3 @@
+(org-habit-stats :repo "ml729/org-habit-stats"
+                 :fetcher github
+                 :files ("org-habit-stats.el"))
```

Comment:
Also I believe this recipe can be simplified to the default (giving you some wiggle room in the future):
```suggestion
(org-habit-stats :repo "ml729/org-habit-stats" :fetcher github)
```